packages/app/package.json

```diff
@@ -25,6 +25,7 @@
     "@monaco-editor/react": "^4.6.0",
     "@tauri-apps/api": "^1.5.0",
     "@tauri-apps/plugin-log": "https://github.com/tauri-apps/tauri-plugin-log#v1",
+    "tauri-plugin-channel-api": "https://github.com/fourviere/tauri-plugin-channel#sender_receiver_ts",
```

Comment:
MR in plugin is approved can you please change it to main when merged?

Reply:
I was thinking of using the v0.1.x branch where only bugfix updates will be posted
The next minor version step will take place with the release of Tauri 2.0